src/router/index.js

```diff
@@ -56,16 +56,25 @@ export default new Router({
       ],
     },
     {
-      path: '/pages/not-found-advanced',
-      component: VuesticPageNotFoundSearch
-    },
-    {
-      path: '/pages/not-found-simple',
-      component: VuesticPageNotFoundSimple
-    },
-    {
-      path: '/pages/not-found-custom',
-      component: VuesticPageNotFoundCustom
+      path: '/pages',
```

Comment:
👍 

You can also have default route within /pages which would redirect to `not-found-simple` for example